src/brod_group_coordinator.erl

```diff
@@ -462,6 +462,15 @@ terminate(Reason, #state{ connection = Connection
 
 %%%_* Internal Functions =======================================================
 
+% Exit the process if MemberPid is set to pid but it's not
+% currently alive
+-spec ensure_member_pid_alive(pid()) -> ok.
+ensure_member_pid_alive(MemberPid) ->
+  case brod_utils:is_pid_alive(MemberPid) of
+    true -> ok;
+    false -> exit(member_pid_shutdown)
```

Comment:
should exit with `{shutdown, Reason}` otherwise there is still going to be a crash log.

Reply:
That makes sense to me. I'll modify the PR.